README.md

```diff
@@ -26,6 +26,4 @@ For developers:
 Using
 -----
 
-Documentation (built nightly) is available here:
-
-http://ssb.stsci.edu/doc/jwst_dev/
+[![Documentation Status](https://readthedocs.org/projects/jwst-pipeline/badge/?version=latest)](http://jwst-pipeline.readthedocs.io/en/latest/?badge=latest)
```

Comment:
There are a lot of people at STScI and around the world who've been told for the past couple years to find our docs at http://ssb.stsci.edu/doc/jwst_dev/, so this change is going to cause a lot of bookmarks and links to break, and also cause a flood of sudden help calls. Is there some way to keep both URL's active for a while to allow for a transition period? I don't want jwst_dev to suddenly just be gone or obsolete suddenly as of today. A change like this really needs to be coordinated before implementing.

Reply:
Of course, that is understandable. It is easy to have http://ssb.stsci.edu/doc/jwst_dev/ re-direct to RTD URL as we have full control of `ssb.stsci.edu`. Alternately, we can provide both links but then we have to explain how one is nightly (which might or might not be slightly outdated compared to `master`) and one is on-the-fly (give or take 30 mins to build, if no error).